queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name ILIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN DATE '2016-01-01' AND '2019-01-01';
+SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = 'true';
```

Comment:
Please add the correct query and sreenshot of the result 🤗🛠

![image](https://github.com/Paulinelali/vet-clinic-project/assets/78251927/0eadc2df-78a7-407d-a830-9c5fdf3f18a7)